charts/prometheus/Chart.yaml

```diff
@@ -26,6 +26,18 @@ engine: gotpl
 type: application
 dependencies:
   - name: kube-state-metrics
-    version: "3.1.*"
+    version: "3.4.*"
     repository: https://prometheus-community.github.io/helm-charts
     condition: kubeStateMetrics.enabled
+  - name: prometheus-node-exporter
+    version: "2.0.*"
+    repository: https://prometheus-community.github.io/helm-charts
+    condition: nodeExporter.enabled
+  - name: prometheus-pushgateway
+    version: "1.10.*"
+    repository: https://prometheus-community.github.io/helm-charts
+    condition: pushgateway.enabled
+  - name: alertmanager
+    version: "1.12.*"
```

Comment:
Error: Error linting charts: Error building dependencies for chart 'prometheus => (version: "14.5.0", path: "charts/prometheus")': Error waiting for process: exit status 1
Error: can't get a valid version for repositories alertmanager. Try changing the version constraint in Chart.yaml

Reply:
Sorry, it was typo.
Fixed